packages/jest-snapshot/src/__tests__/__snapshots__/printSnapshot.test.ts.snap

```diff
@@ -375,6 +375,24 @@ Snapshot: <m>"delete"</>
 Received: <t>"insert"</>
 `;
 
+exports[`printSnapshotAndReceived ignore indentation ignores custom indentation of unchanged line 1`] = `
+<m>- Snapshot  - 2</>
+<t>+ Received  + 2</>
+
+<m>-   **some plugin-generated string with custom leading whitespace**</>
+<t>+   **some plugin-generated string with custom leading whitespace**</>
```

Comment:
this should not be detected as changed because it's exactly the same line in both snapshot and received values